### PR TITLE
Update 4.md

### PR DESCRIPTION
**Pull request by com-master**
Make storage better

### CODE REVIEW

File: src/Standard/TIP-4/4.md [hunk 1]
```diff
@@ -84,39 +84,49 @@ Every `Storage` contract must implement `TIP4_4Storage`
 pragma ton-solidity >= 0.58.0;
 
 interface TIP4_4Storage {
-    function fill(uint8 id, bytes chunk, address gasReceiver) external;
+    function fill(uint32 id, bytes chunk) external;
```
Comment (ghost) [inline src/Standard/TIP-4/4.md:87]: `uint8` > `uint32` is good ✅

Removing `address gasReceiver` is bad ❌
`gasReceiver` is required for internal messages. Need to know where to send remaining gas after each transaction. Example:
* Balance of `Storage` contract is **0.1 EVER**
* Call `fill()` with **1 EVER** using internal message **first time**. **0.1 EVER** spend for fee fee. **0.9 EVER** back to `gasReceiver`
* Call `fill()` with **1 EVER** using internal message **last time**. **0.1 EVER** spend for fee. **0.9 EVER** back to `gasReceiver`
* Balance of `Storage` contract still is **0.1 EVER**

`gasReceiver` is also required for external messages. Need to know where to send remaining gas from `Storage` contract balance in last `fill()` call. Example:
* Balance of `Storage` contract is **1 EVER**
* Call `fill()` with using external message **first time**. **0.1 EVER** spend for fee fee. * Balance of `Storage` contract is **0.9 EVER**
* Call `fill()` with using external message **last time**. **0.1 EVER** spend for fee. **0.7 EVER** form `Storage` contract back to `gasReceiver`
* Balance of `Storage` contract is **0.1 EVER**

File: src/Standard/TIP-4/4.md [hunk 2]
```diff
@@ -84,39 +84,49 @@ Every `Storage` contract must implement `TIP4_4Storage`
...
     function getInfo() external view responsible returns (
         address nft,
         address collection,
         string mimeType,
-        mapping(uint8 => bytes) content
+        bool zstdEncoding,
```
Comment (ghost) [inline src/Standard/TIP-4/4.md:92]: I think `bool` value is wrong. Compression method may differ from zstd. In my opinion, a good solution would be to use a `string`, as is done in [HTTP header Content-Encoding](https://developer.mozilla.org/ru/docs/Web/HTTP/Headers/Content-Encoding)

**Review by ghost — CHANGES_REQUESTED** [review]
Proposals:
* Move back `gasReceiver`
* Use `string encoding`

**Comment by com-master** [comment]
Was moved to https://github.com/nftalliance/docs/pull/34

**Pull request closed without merging**